src/compiler/clvm.rs

```diff
@@ -228,9 +228,9 @@ pub fn convert_to_clvm_rs(
                     })
             }
         }
-        SExp::Cons(_, a, b) => convert_to_clvm_rs(allocator, a.clone()).and_then(|head| {
+        SExp::Cons(_, a, b) => convert_to_clvm_rs(allocator, a.clone()).and_then(|head_ptr| {
```

Comment:
Renamed because it shadowed the function parameter `head`, which is what is actually intended to be printed below.